cmd/edenSetup.go

```diff
@@ -71,6 +75,7 @@ func newSetupCmd(configName, verbosity *string) *cobra.Command {
 	setupCmd.Flags().BoolVarP(&cfg.Adam.APIv1, "api-v1", "", cfg.Adam.APIv1, "use v1 api")
 
 	setupCmd.Flags().IntVar(&cfg.Eve.ImageSizeMB, "image-size", defaults.DefaultEVEImageSize, "Image size of EVE in MB")
+	setupCmd.Flags().StringVar(&imageFormat, "format", "", "format for image, one of 'container','qcow2','raw','qcow','vmdk','vhdx'; if not provided, defaults to container image for docker and oci transports, qcow2 for file and http/s transports")
```

Comment:
Had to add it, because we were actually using it in this function but did not set it

Reply:
As I can see we use it only in utils.EVEDescription{}, correct? If so, please remove, it is not required for `DownloadUEFI` function.

Reply:
We use imageFormat within SDN inside SetupEden -> setupSdn

Reply:
Got it. Inside `setupSdn` imageFormat was used in `utils.DownloadUEFI` which is not required. Fixed that